keeper-gradle-plugin/src/main/java/com/slack/keeper/InferAndroidTestKeepRules.kt

```diff
@@ -33,9 +33,9 @@ import org.gradle.kotlin.dsl.repositories
 import java.util.Locale
 
 /**
- * Generates proguard keep rules from the generated [androidTestJar] and [appJar] tasks,
- * where the generates rules are based on what classes from [appJar] are used by
- * [androidTestJar].
+ * Generates proguard keep rules from the generated [androidTestSourceJar] and [appTargetJar] tasks,
+ * where the generates rules are based on what classes from [appTargetJar] are used by
+ * [androidTestSourceJar].
```

Comment:
Was there a reason these names needed to change?

Reply:
No strong reasons but after adding a new parameter, it was difficult to find an "alternative" naming for the task.
Also when I read the code, it's always a challenge to match extension -> task fields.
So I'm fine reverting renaming but why not also consider just align both naming + use the this. prefix prefix in the builder function? It would increase readability IMHO

Reply:
Naming change is fine, we'll just need to remember to mention in the notes.

Reply:
You're right, because it's part of the task API.